Automattic/wp-calypso
# Checkout: Update discount message when upgrading from domain to plan

Part of #11480. ~~Requires backend patch `D4628-code`.~~

We currently show this notice below the cart details to indicate the discount:

> You're saving $20! Your recent domain purchase is deducted from the price ($35.88), since the plan includes a free domain.

This is not actually right, since the discount is not related to the price of the domain. The actual details of how the discount is applied are a bit too complicated to go over in this notice. For consistency, probably best to just repeat the same message from the plan selection screen.

Also worth covering the renewal price here, to save any unpleasant surprises down the road.

> You're getting a $888 discount off the regular price of the plan ($888), because you already paid for the domain. 
> 
> The plan and the domain can be renewed together for $888 / year. 

After changes:

![screen shot 2017-03-07 at 12 54 46](https://cloud.githubusercontent.com/assets/699132/23655441/4856fa40-0335-11e7-9854-c3001bba4d2b.png)

### Testing
1. ~~Apply backend patch `D4628-code`.~~
2. Purchase domain using domain only site flow.
3. Go to My Sites.
4. Click `Create Site` button.
5. Select plan and theme.
6. You should see checkout page with the updated discount copy.
![screen shot 2017-03-07 at 12 54 46](https://cloud.githubusercontent.com/assets/699132/23655441/4856fa40-0335-11e7-9854-c3001bba4d2b.png)

### Review
